generate-docs.js

```diff
@@ -20,22 +20,30 @@ const isHtmlFile = fileName => fileName.endsWith('.html');
 
 const pipe = (...fns) => start => fns.reduce((state, fn) => fn(state), start);
 
-function hideSearchbar() {
+function adjustmentForGitHubPages() {
   const paths = flatten(readDir('./documentation')).filter(isHtmlFile);
-  paths.forEach(path => replaceSearchbar(path));
-}
+  paths.forEach(path => replaceUnderlinePrefixAndHtmlSuffixFromLinks(path));
```

Comment:
as discussed add a little explainantion with your four link examples in the description so that one quickly understands the reason for the adjustment.

Reply:
done